android/app/build.gradle

```diff
@@ -195,7 +195,7 @@ dependencies {
 
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
     implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
-
+    
```

Comment:
You should before committing and pushing check that the changes you did are intended.
This change seems redundant next time pass over the changes and if they are not intended just reset the changes.